docs/contracts/overview/KeyManager.md

```diff
@@ -21,6 +21,28 @@ To illustrate, in the case of Universal Profile and the browser extension, you c
 
 Afterwards, you can grant partial access to dApp and protocol (more specifically the smart contract addresses behind them), so that they can use your Universal Profile to interact on your behalf. This catalyze interactions on the blockchain, where your Universal Profile can do more and operate automatically in controlled manner, without requiring the main profile owner to have to do everything.
 
+### Allowed Calls
+
+:::note
+
+If controller has `CALL` permission but has no `AllowedCalls`, controller will not be able to use the `CALL` permission at all. 
+
+:::
+
+Key Manager allows for restricting controller addresses to be able to call specific functions on specific addresses which should be of a specific standard. In order to achieve such functionallity one could encode a set of type calls, addresses, standards and functions to [`bytes32[CompactBytesArray]`](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#bytesncompactbytesarray), and store them under the [Allowed Calls](../../standards/universal-profile/lsp6-key-manager.md#allowed-calls) data key.
+
+_E.g._
+
+Supposedly we have the following `AllowedCalls`:
+`0x002000000002cafecafecafecafecafecafecafecafecafecafe24871b3d7f23690c002000000003cafecafecafecafecafecafecafecafecafecafe24871b3d44c028fe`
+
+It can be decoded as:
+
+| Allowed Calls | |
+|:-----|:-----|
+| First allowed call | Call Types - `0x00000002` (Call) <br/> Address - `0xcafecafecafecafecafecafecafecafecafecafe` <br/> Standard - `0x24871b3d` (LSP0) <br/> Function - `0x7f23690c` (`setData(bytes32,bytes)`) |
+| Second allowed call | Call Types - `0x00000003` (Transfervalue & Call) <br/> Address - `0xcafecafecafecafecafecafecafecafecafecafe` <br/> Standard - `0x24871b3d` (LSP0) <br/> Function - `0x44c028fe` (`execute(uint256,address,uint256,bytes)`) |
+
```

Comment:
Maybe you can add here a sentence after the table to explain that in these cases the user is allowed to call and transfer the address cafecafe if it supports the standard `0x24871b3d` using ERC165 and can only interact with the execute function ..
Something like this